src/main/java/com/meetup/server/event/dto/response/RouteResponse.java

```diff
@@ -40,6 +40,19 @@ public static RouteResponse of(StartPoint startPoint,
                                    User user,
                                    OdsayTransitRouteSearchResponse transitResponse,
                                    KakaoMobilityResponse drivingResponse) {
+
+        List<DrivingRouteResponse> drivingRouteResponseList = DrivingRouteResponse.from(drivingResponse);
+
+        int driveTime = 0;
+        if (drivingRouteResponseList != null && !drivingRouteResponseList.isEmpty() && drivingRouteResponseList.get(0) != null) {
+            driveTime = drivingRouteResponseList.get(0).duration();
+        }
+
+        int transitTotalTime = 0;
+        if (transitResponse != null && transitResponse.data() != null && transitResponse.data().path() != null && !transitResponse.data().path().isEmpty()) {
```

Comment:
```suggestion
int transitTotalTime = Optional.ofNullable(transitResponse)
                .map(OdsayTransitRouteSearchResponse::data)
                .map(OdsayTransitRouteSearchResponse.TransitData::path)
                .filter(paths -> !paths.isEmpty())
                .map(path -> path.getFirst().info().totalTime())
                .orElseThrow(() -> new EventException(EventErrorType.NOT_FOUND_ROUTE_INFO));
```

Optional을 사용해서 null 검증을 좀 더 깔끔하게 짤 수도 있어요!
그리고 경로가 없을 경우에 에러를 던지는건 어떨까요?

Reply:
경로가 없는 경우에 에러를 던지면 지도 조회가 안될 것 같아 고민이 되네요 이런 경우에도 에러 던지는 게 좋을 지 빈 값으로 넘기는 방법 중에 어떤게 더 좋을까요?

Reply:
만약 경로가 나오지 않는다면 그거는 외부 API의 문제일 수도 있을 거 같긴 하네요.!

외부 api에 의존하는 서비스인 만큼, 예외 케이스는 같이 고민해봐요!

우선은 예외 안던지는 걸로 가도 괜찮을 거 같습니다